### ISSUE TITLE
Depict transport movements in campus

### ISSUE THREAD

**Issue by saarzhanova** [saarzhanova/Campus3DMap#6]
Based on the data from Ils-de-France Mobilite, show the paths of public transport movements around the campus with lines.

= Ask for the data about the buildings, Ils-de-France Mobilite data

**Comment by saarzhanova** [comment]
Need to find the data from Ils-deFrance Mobilite about public transport movements around the campus here: https://data.iledefrance-mobilites.fr/explore/?disjunctive.theme&disjunctive.keyword&disjunctive.publisher&disjunctive.modified&disjunctive.features&disjunctive.language&sort=modified

**Comment by saarzhanova** [comment]
Based on traces-des-lignes-de-transport-en-commun-idfm.json" from [Ils-de-France Mobililte](https://data.iledefrance-mobilites.fr/explore/dataset/traces-des-lignes-de-transport-en-commun-idfm/table/?disjunctive.route_type) I depicted campus bus traces.

https://github.com/saarzhanova/Campus3DMap/commit/83da2149b0d2c9975c8322ab4949188d9cd2cca0

The problem is that I need to find out how to attach the coordinates of bus traces to coordinates on the campus model. For the moment I attached them manually. It is still to be precised.
